GYBITG/Base.lproj/Main.storyboard

```diff
@@ -92,68 +92,104 @@
+                            <constraint firstItem="qN8-me-8ih" firstAttribute="centerX" secondItem="ohI-EW-ROe" secondAttribute="centerX" id="F35-Qe-h21"/>
+                            <constraint firstItem="Yri-3d-hrL" firstAttribute="top" secondItem="qN8-me-8ih" secondAttribute="bottom" constant="28" id="Htp-Da-otx"/>
+                            <constraint firstItem="i4i-eX-Fe0" firstAttribute="trailing" secondItem="rGB-cO-yXL" secondAttribute="trailing" id="JAg-q8-sSW"/>
+                            <constraint firstItem="OLb-cT-lcb" firstAttribute="top" secondItem="jiQ-vx-Ge2" secondAttribute="bottom" constant="8" symbolic="YES" id="U5H-RB-mHL"/>
+                            <constraint firstItem="rGB-cO-yXL" firstAttribute="top" secondItem="i4i-eX-Fe0" secondAttribute="bottom" constant="8" symbolic="YES" id="a9M-t5-ert"/>
+                            <constraint firstItem="qN8-me-8ih" firstAttribute="leading" secondItem="O1u-W8-tvY" secondAttribute="leading" constant="37" id="dr6-oA-9qw"/>
+                            <constraint firstItem="i4i-eX-Fe0" firstAttribute="top" secondItem="uLi-pb-8FA" secondAttribute="bottom" constant="1" id="e03-9W-qvQ"/>
+                            <constraint firstItem="OLb-cT-lcb" firstAttribute="leading" secondItem="i4i-eX-Fe0" secondAttribute="leading" id="g0R-YN-heI"/>
+                            <constraint firstItem="rGB-cO-yXL" firstAttribute="leading" secondItem="O1u-W8-tvY" secondAttribute="leading" constant="238" id="hga-eg-09N"/>
+                            <constraint firstItem="qN8-me-8ih" firstAttribute="centerX" secondItem="OLb-cT-lcb" secondAttribute="centerX" id="mRG-ml-Fay"/>
+                            <constraint firstItem="qN8-me-8ih" firstAttribute="top" secondItem="O1u-W8-tvY" secondAttribute="top" constant="29" id="mm3-KO-HcG"/>
+                            <constraint firstItem="qN8-me-8ih" firstAttribute="centerX" secondItem="i4i-eX-Fe0" secondAttribute="centerX" id="msN-G7-sOe"/>
+                            <constraint firstItem="qN8-me-8ih" firstAttribute="centerX" secondItem="OKK-Yi-t2O" secondAttribute="centerX" id="tI6-dZ-dF6"/>
+                            <constraint firstItem="OKK-Yi-t2O" firstAttribute="leading" secondItem="ohI-EW-ROe" secondAttribute="leading" id="uWZ-ji-tJd"/>
+                            <constraint firstItem="uLi-pb-8FA" firstAttribute="leading" secondItem="O1u-W8-tvY" secondAttribute="leading" constant="79" id="uZl-9G-pmo"/>
+                            <constraint firstItem="OKK-Yi-t2O" firstAttribute="top" secondItem="rGB-cO-yXL" secondAttribute="bottom" constant="35" id="wHZ-DZ-pca"/>
+                        </constraints>
                         <viewLayoutGuide key="safeArea" id="O1u-W8-tvY"/>
                     </view>
-                    <tabBarItem key="tabBarItem" title="" image="icons-female-profile-50" id="cPa-gy-q4n"/>
```

Comment:
I think it was this line that caused the error. Try changing the id to get a different id. This should make the error of more than 1 id are same go away.

Reply:
I commented out line 611, in Main.Storyboard: <tabBarItem key="tabBarItem" title="Account" image="second" id="cPa-gy-q4n"/> and the app built.

Reply:
I just commented out line 611 and it's building now.